bundle/Makefile

```diff
@@ -1,12 +1,18 @@
 PREFIX ?= $(DESTDIR)/usr/local
 ETCDIR ?= $(DESTDIR)/etc
+CONTAINERS_DIR ?= $(ETCDIR)/containers
+CNIDIR ?= $(ETCDIR)/cni/net.d
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 OCIDIR ?= $(PREFIX)/share/oci-umount/oci-umount.d
 SELINUX ?= $(shell selinuxenabled 2>/dev/null && echo -Z)
 BASHINSTALLDIR ?= $(PREFIX)/share/bash-completion/completions
 FISHINSTALLDIR ?= $(PREFIX)/share/fish/completions
 ZSHINSTALLDIR ?= $(PREFIX)/share/zsh/site-functions
+SYSTEMDDIR ?= $(PREFIX)/lib/systemd/system
+CRIOWIPEDIR ?= $(PREFIX)/libexec/crio/crio-wipe
```

Comment:
this is no longer needed as of https://github.com/cri-o/cri-o/pull/2746

Reply:
Yep, dropped it